config/docker_settings.rb

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 server_url ::ENV['CHEF_SERVER_URL']
 client_name ::ENV['CHEF_CLIENT_NAME']
 client_key ::ENV['CHEF_CLIENT_KEY']
 connection[:ssl] = { verify: false } if ::ENV['INSECURE_SSL']
 title ::ENV['TITLE']
 use_partial_search !::ENV['NO_PARTIAL_SEARCH']
-login !!::ENV['LOGIN']
+login ::ENV['LOGIN']
```

Comment:
This value is later properly evaluated to true/false, so we don't have to do double negation here.